From a271effaaf8bcaf86f8175e5ad96969757d4c4fd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <cynerd@email.cz>
Date: Wed, 12 Aug 2020 16:35:07 +0200
Subject: [PATCH] syncthing: move home and logfile directories to /srv

Turris uses /srv and has support for easy mounting of drives to that
location.

[rebase, move logfile directory as well, update commit message
accordingly]
Signed-off-by: Šimon Bořek <simon.borek@nic.cz>
---
 utils/syncthing/files/syncthing.conf | 4 ++--
 utils/syncthing/files/syncthing.init | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/utils/syncthing/files/syncthing.conf b/utils/syncthing/files/syncthing.conf
index 38dac9b77..6d98cfc8f 100644
--- a/utils/syncthing/files/syncthing.conf
+++ b/utils/syncthing/files/syncthing.conf
@@ -8,7 +8,7 @@ config syncthing 'syncthing'
 	# production. This filesystem must either support ownership/attributes or
 	# be readable/writable by the user specified in 'option user'.
 	# Consult syslog if things go wrong.
-	option home '/etc/syncthing'
+	option home '/srv/syncthing'
 
 	# Changes to "niceness"/macprocs are not picked up by "reload_config" or by
 	# "restart": the service has to be stopped/started for those to take effect.
@@ -26,7 +26,7 @@ config syncthing 'syncthing'
 	# Running as 'root' is possible, but not recommended
 	# option user 'syncthing'
 
-	option log_file '/var/log/syncthing.log'
+	option log_file '/srv/syncthing/syncthing.log'
 	# option log_level 'INFO'
 	option log_max_old_files '7'
 	# Size in bytes
diff --git a/utils/syncthing/files/syncthing.init b/utils/syncthing/files/syncthing.init
index b50abf74f..c6d4a4661 100644
--- a/utils/syncthing/files/syncthing.init
+++ b/utils/syncthing/files/syncthing.init
@@ -43,8 +43,8 @@ start_service() {
 	local db_maintenance_interval=''
 	local gui_address='http://0.0.0.0:8384'
 	local gui_apikey=''
-	local home='/etc/syncthing'
-	local log_file='/var/log/syncthing.log'
+	local home='/srv/syncthing'
+	local log_file='/srv/syncthing/syncthing.log'
 	local log_level='INFO'
 	local log_max_old_files=7
 	local log_max_size=1048576
-- 
2.52.0

